### ISSUE TITLE
Drop rate higher than 100% in List_NPC.STB ?

### ISSUE THREAD

**Issue by laurenzzo** [eRose-DatabaseCleaning/STB#2]
does it give more items ?
Subquestion : difference between 0 and 'null'

**Comment by hdelanno** [comment]
- There are no differences between '0' and 'null'
- For the drop rate, the formula can be found here : https://github.com/PurpleYouko/Wibble_Wibble/blob/master/Rose_Engine/Client/Common/Calculation.cpp#, in particular see the variable NPC_DROP_ITEM . As you can see this variable affect:
  - drop chances*
  - if you will get a drop from the monster or from the zone
  - chances to have stats, refined item, etc. However it is not a variable that is simple to compute to make stats with since other variables are taken into account.

*Here is how it works:
- you generate a random number between 0 and 100. If it is lower than the "money drop rate" then you have a money drop AND NOTHING ELSE
- then, if you failed to have a money drop, you generate an other number between 0 and 100. If it is lower than the "item drop rate", you will get a "mob item". Else you will get a "map item"

To summarize, the "money drop rate" tells you your chance to have a money or a drop. The "item drop rate" tells you (when you will get an item drop) your chance to get a mob item instead of a map item. A last point it that variable (https://github.com/PurpleYouko/Wibble_Wibble/blob/master/Rose_Engine/Client/Common/Calculation.cpp#L150) has to be positive. Which will be always the case if the server drop rate is high or medium. If it is a really low rate I don't know.
